Makefile

```diff
@@ -46,7 +46,7 @@ exporter-minimal: modcheck
 	env CGO_ENABLED=0 GOOS=$(OS) GOARCH=$(ARCH) go build -o bin/zxp-$(OS)-$(ARCH) -tags minimal,containers_image_openpgp -v -trimpath ./cmd/zxp
 
 .PHONY: test
-test: check-skopeo $(NOTATION)
+test: check-skopeo check-notation
```

Comment:
I am thinking an alternate fix may work.
Can you try `export PATH := bin:$(TOOLSDIR)/bin:$(PATH)` at line 9, instead of the other changes you make in this makefile?